.github/workflows/ruff.yml

```diff
@@ -0,0 +1,8 @@
+name: Ruff
+on: [push, pull_request]
+jobs:
+  ruff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: chartboost/ruff-action@v1
```

Comment:
should this also use the line-length param? maybe it's inferred from the .flake8 ?

Reply:
I think I can remove the `.flake8` file given that I put the following into `pyproject.toml`:
```
[tool.ruff]
line-length = 88
```